components/ActionBar/index.js

```diff
@@ -197,6 +213,12 @@ class ActionBar extends Component {
           {displayConsumptionMinutes > 1 && (
             <ReadingTime minutes={displayConsumptionMinutes} />
           )}
+          {isEditor &&
+            editorIcons
+              .filter(Boolean)
+              .map((props, i) => (
+                <IconLink key={props.icon} fill={fill} {...props} />
```

Comment:
fill comes through props spread:
```suggestion
                <IconLink key={props.icon} {...props} />
```